src/server/info.ts

```diff
@@ -0,0 +1,15 @@
+
+export const repositoryName = 'bookbrainz-site';
+
+export const repositoryUrl = 'https://github.com/metabrainz/bookbrainz-site/';
```

Comment:
Not sure it's particularly necessary, but any reason you didn't use this?
```suggestion
export const repositoryUrl = `https://github.com/metabrainz/${repositoryName}/`;
```

Reply:
I had considered doing this when I moved the constant into this module but then I thought that it only obfuscates the URL without actually simplifying anything.
As long as the constants are declared on consecutive lines, I would rather keep them as is.

Reply:
Little added value indeed.